### ISSUE TITLE
bind/connect sizeof ?

### ISSUE THREAD

**Issue by sgs1370** [mcr/ttcp#1]
First - thank you for maintaining this.  I have been dragging around my own copy until I found this.

I made the modification on my own copy, so if this is not useful comment, just close.

In order to make this work, both on ubuntu 16 and on macos, I had to change the bind and connect functions to have

sizeof(struct sockaddr)

instead of

sizeof(sinme)    (or for connect, sizeof(sinhim))

Otherwise I was getting 

ttcp-t: connect: Invalid argument

or 

ttcp-r: bind: Invalid argument

NOTE: I never did learn how to properly debug C, so I can't say why mine wasn't working in the first place, or why my change works, it could be by accident.  

Anyhow thanks again!!!

**Comment by mcr** [comment]
Yeah, so I think you are exactly correct.  Want to send a pull request?
